### PR TITLE
Fix display form for multiple classes

### PR DESCRIPTION
**Pull request by tsmr**
domain
group
user
operating system

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | none

### CODE REVIEW

File: inc/group.class.php [hunk 1]
```diff
@@ -275,6 +275,7 @@ function showForm($ID, $options = []) {
 
       echo "<tr class='tab_bg_1'>";
       echo "<td class='subheader' colspan='2'>".__('Can contain');
+      echo "</td><td colspan='2' class='center'>";
```
Comment (cedric-anne) [inline inc/group.class.php:278]: Previous row seems to contains only 2 `td`, so existing one using `colspan='2'` was already using the 2 columns.

File: inc/user.class.php [hunk 2]
```diff
@@ -2160,22 +2160,23 @@ function () {
          echo "<td><label for='password'>" . __('Password')."</label></td>";
          echo "<td><input id='password' type='password' name='password' value='' size='20'
                     autocomplete='new-password' onkeyup=\"return passwordCheck();\"></td>";
-         echo "<td rowspan='2'>";
          if ($CFG_GLPI["use_password_security"]) {
```
Comment (cedric-anne) [inline inc/user.class.php:2163]: This is still not correct on my instance:
image

File: inc/group.class.php (hunk 1 above)
Comment (tsmr) [reply inc/group.class.php:278]: Previous rows are linked to rowspan='10' of comment field so we can update to rowspan='13' ?

Comment (trasher) [reply inc/group.class.php:278]: I guess all empty columns should be dropped, and rowspan adapted accordingly. By the way, it seems there are 12 rows total, not 13; and this one is the 10th; so @cedric-anne was right

Comment (tsmr) [reply inc/group.class.php]: I do it

**Pull request merged**